default_crypto.go

```diff
@@ -101,7 +111,7 @@ func (c DefaultCrypto) Encrypt(mk Key, plaintext, ad []byte) []byte {
 	)
 	stream.XORKeyStream(ciphertext[aes.BlockSize:], plaintext)
 
-	return append(ciphertext, c.computeSignature(authKey[:], ciphertext, ad)...)
+	return append(ciphertext, c.computeSignature(authKey[:], ciphertext, ad)...), nil
```

Comment:
This function cannot return any `error`. Is its signature dictated by some interface?

Reply:
yes, this implementation is not used by us, we re-implement the methods, and in our case it might return an error